assets/app/view/game/game_meta.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module View
+  class GameMeta < Snabberb::Component
+    needs :game
+
+    def render
+      children = [h(:h3, 'Game Info')]
+
+      if (publisher = @game.class::GAME_PUBLISHER)
+        children << h(:p, [
+            'Published by ',
+            *Lib::Publisher.link_list(component: self, publishers: Array(publisher)),
+          ])
+      end
+      children << h(:p, "Designed by #{@game.class::GAME_DESIGNER}") if @game.class::GAME_DESIGNER
+      children << h(:p, "Implemented by #{@game.class::GAME_IMPLEMENTER}") if @game.class::GAME_IMPLEMENTER
+      if @game.class::GAME_RULES_URL.is_a?(Hash)
+        @game.class::GAME_RULES_URL.each do |desc, url|
+          children << h(:p, [h(:a, { attrs: { href: url, target: '_blank' } }, desc)])
```

Comment:
i don't think we do target blank anywhere else

Reply:
I didn't add it. I moved this code from game_info.rb > game_info into its own module.